### DISCUSSION TITLE
Just started with breeze evolution

### DISCUSSION THREAD

**Discussion by gfomengia** [breezefront/community#7] [5. General]
Thanks for this great theme. 

I just started trying to implement the breeze evolution theme with third modules already on my site. 

Is there a community like slack, discord etc where there are more developers using breeze? 

Also, it would be nice to see a detailed breakdown of how the breeze theme has been implemented with third-party modules. 

Cheers.

**Comment by vovayatsyuk** [comment]
Hi! Great, I hope there is not too much js code in your existing site :)

There are not so many devs around to create a discord chat I think, but thanks for the suggestion. Right now this place is the only place to talk with other developers.

At the start point, I recommend reading our docs https://breezefront.com.  Especially the parts about js: 

 - https://breezefront.com/components
 - https://breezefront.com/widgets
 - https://breezefront.com/views

Then, take a look at how built-in components are made: 

 - https://github.com/breezefront/module-breeze/tree/master/view/frontend/web/js/components
 - https://github.com/breezefront/module-breeze/blob/master/view/frontend/layout/breeze_default.xml#L61

It's also worth looking at our free modules and how they are integrated with the breeze:

- CSS: https://github.com/swissup/module-easycatalogimg/tree/master/view/frontend/web/css/breeze
- CSS: https://github.com/swissup/module-featured-attributes/tree/master/view/frontend/web/css/breeze
- CSS and JS: https://github.com/swissup/module-easytabs/tree/master/view/frontend
- CSS and JS: https://github.com/swissup/module-testimonials/tree/master/view/frontend
